### DISCUSSION TITLE
Playing an album

### DISCUSSION THREAD

**Discussion by Polibon** [rern/rAudio#1822] [General]
I love rAudio for using the raspberry pi to the full, and letting me get the most out of my system, with full access to my saved music archive. But after hours of 'playing' with the interface on several different devices I still can't figure out how to easily play an album (directory) without adding each track individually to a playlist. Surely there's an easier way? I can't find any tutorials or videos, so have resorted to this. Hopefully there is a button in a menu that I have overlooked. I do love the random selection from my archive; but some albums just need to be listened to in their entirety and in sequence.

**Comment by ccaper** [comment]
Drill into album contents (Library -> Album -> desired album; but there are multiple ways to drill into an album).  Click on the album art image, you will get a context menu (with options 'Add', Icon for Add & Play, if play list has items in it, 'Replace' and Icon for Replace & Play).
